addons/pos_restaurant/static/src/js/models.js

```diff
@@ -324,14 +259,17 @@ patch(PosGlobalState.prototype, "pos_restaurant.PosGlobalState", {
         }
         return new Printer(url, this);
     },
+    interactWithDb() {
+        return this._super(...arguments) || this.config.iface_floorplan;
+    },
 });
 
 // New orders are now associated with the current table, if any.
 patch(Order.prototype, "pos_restaurant.Order", {
     setup(options) {
         this._super(...arguments);
         if (this.pos.config.module_pos_restaurant) {
-            if (this.pos.config.iface_floorplan && !this.tableId && !options.json) {
+            if (this.pos.config.iface_floorplan && !this.tableId && !options.json && this.pos.table) {
```

Comment:
What's the case where you don't have a table and the order doesn't have a table as well ? I guess it's for the refund orders coming from a normal point_of_sale ?